test/mix/tasks/phx.gen.auth_test.exs

```diff
@@ -192,16 +192,16 @@ defmodule Mix.Tasks.Phx.Gen.AuthTest do
 
       assert_file("lib/my_app_web/components/layouts/root.html.heex", fn file ->
         assert file =~
-                 ~S|<.link href={~p"/users/settings"}>Settings</.link>|
```

Comment:
Note that I've killed off the inner text in these assertions due to mess it'd have to make of the regexp matchers due to the additional whitespace added by the formatter. I'm comfortable with the removal of the text and closing-tag from these assertions due to their being covered in the snapshot tests in this file and the injector tests, both of which due 1:1 string equality checks on the entirety of the output.